app/models/health_care_application.rb

```diff
@@ -272,9 +272,9 @@ def log_submission_failure_details
       'HCA total failure',
       :error,
       {
-        first_initial: parsed_form['veteranFullName']['first'][0],
-        middle_initial: parsed_form['veteranFullName']['middle'].try(:[], 0),
-        last_initial: parsed_form['veteranFullName']['last'][0]
+        first_initial: parsed_form.dig('veteranFullName', 'first')&.[](0) || 'no initial provided',
```

Comment:
Updated these to use `dig` so that if for some reason they are not there it will display `no initial provided` instead of just being empty or throwing an error.